setup.py

```diff
@@ -24,7 +24,7 @@ def read_file(filename):
     "cornice",
     "cornice_swagger >= 0.5.1",
     "dockerflow",
-    "jsonschema",
+    "jsonschema >= 2.9.9",
```

Comment:
I wish we could come up with a better way of naming this.
```suggestion
    "jsonschema >= 3",
```